samples/dapr/.rad/rad.yaml

```
@@ -0,0 +1,2 @@
+workspace:
+  application: "dapr-app"
```

The application name in the bicep definition is just `dapr`

```suggestion
  application: "dapr"
```